ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -524,6 +520,10 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         tokenSet.registerOnUpdate(for: view) { [weak self] in
             self?.updateBackgroundColor()
         }
+
+        if #available(iOS 16, *) {
+            setNeedsUpdateOfSupportedInterfaceOrientations()
```

Comment:
do we need this here? if we do need to notifiy it should be when we rotate not in viewdidload?

Reply:
Looking at the documentation it seems it needs to be called only when either of these variables `supportedInterfaceOrientations` or `preferredInterfaceOrientationForPresentation` changes. So I'm not sure if it would make sense for this to be called when rotation changes unless I'm misunderstanding something.